mandelbrot_chaco.py

```diff
@@ -49,8 +49,10 @@ def dispatch(self, event, suffix):
 
 
 class MandelbrotPlot(HasStrictTraits):
-    ranges = Array(dtype=Float, value=[-2, 1, -1.5, 1.5])
+    _default_ranges = [-2, 1, -1.5, 1.5]
+    ranges = Array(dtype=Float, value=_default_ranges)
     plot = Property(depends_on='ranges')
+    button = Button("Reset of Panning and Zooming")
```

Comment:
The `Button` trait is convenient, and it works fine in this application.  For anything larger, I would want to see this pattern:
```
class MandelbrotPlot(HasStrictTraits):
    # ...
    reset_zoom = Event
    # ...
    @observe('reset_zoom')
    def reset_zoom(self, event):
        # ...
    traits_view = View(
        # ...
        UItem('reset_zoom', editor=ButtonEditor(label="Reset Zoom")),
    )
```
This decouples the view aspects from the model, which is better for testing, etc.  It's probably too much complexity for this simple case, but in principle, I'd prefer to see an `Event` trait modified by a `ButtonEditor`.

Reply:
@timdiller 
Thank you for your comment. This is very helpful. I modified the code. https://github.com/kuromusha/mandelbrot-chaco/pull/3/commits/f240dd70b71eb481816172546607addf73a90a2e